launch/ntrip_client.launch

```diff
@@ -60,7 +60,7 @@
     <param name="rtcm_frame_id" value="odom" />
 
     <!-- Optional parameters that will allow for longer or shorter NMEA messages. Standard max length for NMEA is 82 -->
-    <param name="nmea_max_length" value="83" />
+    <param name="nmea_max_length" value="84" />
```

Comment:
Please add in a comment here about why we're doing this. And the max byte sizes for both.
NMEA --> 82
High precision NMEA --> 84(?)

Reply:
I have added a comment